add-tourpackage.php

```diff
@@ -191,67 +182,67 @@
                     <div class="col-12 mt-3">
                         <div class="row">
                             <div class="col-12 col-md-4 col-lg-4">
-                                <label>Total Milage</label>
+                                <label>Total Milage (KM)</label>
```

Comment:
I recommend that the proper way to add units (Km, m, cm, etc.) is to store the table in the database. Because if you hard-code these parameters, you have to check and update all the process files one by one in the update requirement.

Reply:
Will try to do.